src/job.proto

```diff
@@ -24,6 +25,19 @@ message JobResult {
   }
 }
 
+message EsdlMessage {
```

Comment:
I might consider reordering it as

severity
esdl_object_id
technical_message